packages/fluent/scss/stepper/_variables.scss

```diff
@@ -0,0 +1,197 @@
+@use "../core/_variables" as *;
+@use "../core/color-system" as *;
+@use "../core/mixins" as *;
+
+/// Width of the border around the stepper.
+/// @group stepper
+$kendo-stepper-border-width: 0px !default;
+// Font size of the stepper.
+$kendo-stepper-font-size: var( --kendo-font-size, inherit ) !default;
+// Font family of the stepper.
+$kendo-stepper-font-family: var( --kendo-font-family, inherit ) !default;
+// Line height of the stepper.
+$kendo-stepper-line-height: var( --kendo-line-height, normal ) !default;
+
+/// Background color of the stepper.
+/// @group stepper
+$kendo-stepper-bg: transparent !default;
+/// Text color of the stepper.
+/// @group stepper
+$kendo-stepper-text: var( --kendo-component-text, initial ) !default;
+/// Border color of the stepper.
+/// @group stepper
+$kendo-stepper-border: transparent !default;
+
+/// Horizontal margin the stepper label.
+/// @group stepper
+$kendo-stepper-label-margin-x: map-get( $kendo-spacing, 2 ) !default;
+/// Horizontal padding the stepper label.
+/// @group stepper
+$kendo-stepper-label-padding-x: map-get( $kendo-spacing, 3 ) !default;
+/// Vertical padding the stepper label.
+/// @group stepper
+$kendo-stepper-label-padding-y: calc( map-get( $kendo-spacing, 1 ) + map-get( $kendo-spacing, thin ) ) !default;
```

Comment:
isn't the padding 4px in the design? It's 5px including the border

Reply:
Actually stepper label does not have a border by design. It has only 2px outline on focus.